### PR TITLE
FlipperDevicesInc.qFlipper version 1.3.0.0

### PR DESCRIPTION
**Pull request by krishisrani61**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/102558)

### CODE REVIEW

**Comment by israniboi** [comment]
@microsoft-github-policy-service agree company="Flipper Devices Inc
"

**Comment by israniboi** [comment]
@microsoft-github-policy-service agree

File: manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: FlipperDevicesInc.qFlipper
+PackageVersion: 1.3.0.0
```
Comment (mdanish-kh) [inline manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.installer.yaml:5]:
```suggestion
PackageVersion: 1.3.0
```
Must match Apps&Features / Control Panel

File: manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: FlipperDevicesInc.qFlipper
+PackageVersion: 1.3.0.0
```
Comment (mdanish-kh) [inline manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 1.3.0
```
Must match Apps&Features / Control Panel

File: manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: FlipperDevicesInc.qFlipper
+PackageVersion: 1.3.0.0
```
Comment (mdanish-kh) [inline manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.yaml:5]:
```suggestion
PackageVersion: 1.3.0
```
Must match Apps&Features / Control Panel

File: manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: FlipperDevicesInc.qFlipper
+PackageVersion: 1.3.0.0
+Installers:
+- Architecture: neutral
```
Comment (mdanish-kh) [inline manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.installer.yaml:7]:
```suggestion
- Architecture: x64
```
Neutral means the installer supports all architectures (even the arm ones I believe). Looking at the installer URL and the architecture from Task Manager, I don't think this is the case.
image

File: manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: FlipperDevicesInc.qFlipper
+PackageVersion: 1.3.0.0
+Installers:
+- Architecture: neutral
+  InstallerType: nullsoft
+  NestedInstallerType: nullsoft
```
Comment (mdanish-kh) [inline manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.installer.yaml:9]:
```suggestion
```
The installer is not compressed into a zip archive, so this field is not needed here

File: manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: FlipperDevicesInc.qFlipper
+PackageVersion: 1.3.0.0
+Installers:
+- Architecture: neutral
+  InstallerType: nullsoft
+  NestedInstallerType: nullsoft
+  InstallerUrl: https://update.flipperzero.one/builds/qFlipper/1.3.0/qFlipperSetup-64bit-1.3.0.exe
+  InstallerSha256: 1B4BEF70AD21359E5AA36B6C9E875C9BD158BAD11237F07C91145D752C15D208
```
Comment (mdanish-kh) [inline manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.installer.yaml:11]:
```suggestion
  InstallerSha256: 1B4BEF70AD21359E5AA36B6C9E875C9BD158BAD11237F07C91145D752C15D208
  ProductCode: qFlipper
```
Add ProductCode for better upgrade matching.

File: manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: FlipperDevicesInc.qFlipper
+PackageVersion: 1.3.0.0
+Installers:
```
Comment (mdanish-kh) [inline manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.installer.yaml:6]:
```suggestion
Scope: machine
Installers:
```
The package installs machine-wide (for ALL users).

File: manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.locale.en-US.yaml [hunk 8]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: FlipperDevicesInc.qFlipper
+PackageVersion: 1.3.0.0
+PackageLocale: en-US
+Publisher: Flipper Devices Inc.
+PackageName: qFlipper
+License: GPL3
```
Comment (mdanish-kh) [inline manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.locale.en-US.yaml:9]:
```suggestion
License: GPL-3.0
LicenseUrl: https://github.com/flipperdevices/qFlipper/blob/dev/LICENSE
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
After committing the suggested changes, you would also have to rename the version folder to match the new PackageVersion. The folder under `manifests/f/FlipperDevicesInc/qFlipper` should be renamed from `1.3.0.0` -> `1.3.0`

A quick and easy way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online. Navigate to Explorer menu, find the required folder and make the appropriate rename changes.

After that, stage and commit your changes from Source Control menu on the left. (The changes are automatically pushed)

File: manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.locale.en-US.yaml [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: FlipperDevicesInc.qFlipper
+PackageVersion: 1.3.0.0
+PackageLocale: en-US
+Publisher: Flipper Devices Inc.
+PackageName: qFlipper
+License: GPL3
+Copyright: (C) Flipper Devices Inc.
+ShortDescription: qflipper
```
Comment (mdanish-kh) [inline manifests/f/FlipperDevicesInc/qFlipper/1.3.0.0/FlipperDevicesInc.qFlipper.locale.en-US.yaml:11]:
```suggestion
ShortDescription: qFlipper — desktop application for updating Flipper Zero firmware via PC 
```
ShortDescription should be descriptive

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/102558#pullrequestreview-1386532912

You'll have to modify the directory structure as well

**Comment by stephengillie** [comment]
Automatic Validation failed with:
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\f\FlipperDevicesInc\qFlipper\1.3.0.0', expected 'manifests\f\FlipperDevicesInc\qFlipper\1.3.0' 

There's an extra `.0` in the path.

**Pull request closed without merging**